templates/base.html

```diff
@@ -17,8 +29,16 @@
       content="RTS, real-time, strategy, game, gaming, Rust, free software,
                open source, Linux gaming">
     <meta name="author" content="Martin Indra">
+    <link rel="stylesheet" href="/styles.css">
   </head>
-  <body>
-    {% block content %} {% endblock %}
+  <body style="background-color: #340000;">
```

Comment:
Can we style `body` in the CSS / Sass file?

Reply:
On page load there would sometimes be white page blink and redraw. This does draw page as the correct color on load. :)